root/static/scripts/work.js

```diff
@@ -70,6 +70,7 @@ const store = createStore(function (state: WorkForm = form, action) {
         action.languageId,
         state,
       );
+      state.field.languages.errors = (_.intersection(state.field.languages.field.map(lang => lang.value), ["486", "284"]).length > 0);
```

Comment:
We try to avoid mutating `state` here. I like the idea of the callback function or just passing the `length > 0` check as a boolean.

Note that the type of `lang.value` is `number`, so either `["486", "284"]` doesn't work or they're actually strings and we're not storing it correctly. You can do `String(lang.value)` in the map callback to be sure, though.

Reply:
Great, made relevant changes in new commit.